tools/python/influx-migration/influx_migration.py

```diff
@@ -119,14 +119,15 @@ def report_all_bucket_series_count(host, token, org_name=None):
         while len(buckets.buckets) > 0:
             for bucket in buckets.buckets:
                 if not bucket.name.startswith("_"):
-                    report_bucket_series_count(bucket_name=bucket.name, host=host, token=token, org_name=org_name)
+                    report_bucket_series_count(bucket_name=bucket.name, host=host, token=token,
+                        org_name=org_name, skip_verify=skip_verify)
             offset += BUCKET_PAGINATION_LIMIT
             buckets = client.buckets_api().find_buckets(limit=BUCKET_PAGINATION_LIMIT,
                 offset=offset)
 
-def report_bucket_series_count(bucket_name, host, token, org_name=None):
+def report_bucket_series_count(bucket_name, host, token, org_name=None, skip_verify=False):
```

Comment:
This method is invoked to log the series count in a bucket, but it raises an exception if the count is not found. Why do we need to raise an exception here instead of logging it as an error?

Reply:
If you're referring to the raised `ValueError` within the `try` block, this exception will be caught immediately. I am raising this exception instead of logging the error immediately because I am using everything under the `except` block as a place to log all failures related to being unable to log a bucket's series count, to prevent duplicated code.